server/server.go

```diff
@@ -27,8 +27,13 @@ func newCribbageServer(dbFactory persistence.DBFactory) *cribbageServer {
 	}
 }
 
-func (cs *cribbageServer) NewRouter() http.Handler {
-	router := gin.Default()
+func (cs *cribbageServer) NewRouter(useDefaultMiddleware bool) http.Handler {
+	var router *gin.Engine
+	if useDefaultMiddleware {
+		router = gin.Default()
+	} else {
+		router = gin.New()
+	}
```

Comment:
Okay, so I did this to make the test output less verbose... But I'm not sure if I'm in love with it. Thoughts @joshprzybyszewski ?

Reply:
I'd rather keep the verbose test output:#